samples/excel/90-just-for-fun/patterns.yaml

```diff
@@ -0,0 +1,210 @@
+id: excel-90-just-for-fun-patterns-
+name: Colorful Patterns
+description: Shows how to use range formatting to draw interesting pattern. Contributed by Alexander Zlatkovski
+author: AlexanderZlatkovski
+host: EXCEL
+api_set:
+    ExcelApi: 1.2
+script:
+    content: |-
+        $("#spiral").click(drawSpiral);
+        $("#squares").click(drawSquares);
+        $("#cross").click(drawCross);
+
+        async function drawSquares() {
+            try {
+                await Excel.run(async (context) => {
+                    const sheet = await OfficeHelpers.ExcelUtilities.forceCreateSheet(context.workbook, "Patterns", true /*clearOnly*/);
+                    sheet.activate();
```

Comment:
Creating a new sheet and activating it with the same context.sync() will cause an issue in Excel Online. Insert a context.sync() right before sheet.activate().